mlflow/genai/evaluation/harness.py

```diff
@@ -224,6 +231,11 @@ def _log_assessments(
                 **(assessment.metadata or {}),
                 AssessmentMetadataKey.SOURCE_RUN_ID: run_id,
             }
+
+        # NB: Root span ID is necessarily to show assessment results in DBX eval UI.
+        if root_span := trace.data._get_root_span():
```

Comment:
I remember encounter this error sometimes that assessment doesn't have span_id during bug bash. Could we add a debug log if this doesn't exist?